<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZBodyListWidget</class>
 <widget class="QWidget" name="ZBodyListWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Body IDs</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="addPushButton">
       <property name="text">
        <string>Add</string>
       </property>
       <property name="icon">
        <iconset resource="../gui.qrc">
         <normaloff>:/images/add.png</normaloff>:/images/add.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deletePushButton">
       <property name="text">
        <string>Delete</string>
       </property>
       <property name="icon">
        <iconset resource="../gui.qrc">
         <normaloff>:/images/delete.png</normaloff>:/images/delete.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="ZFlyEmBodyListView" name="listView"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ZFlyEmBodyListView</class>
   <extends>QListView</extends>
   <header>flyem/zflyembodylistview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../gui.qrc"/>
 </resources>
 <connections/>
</ui>
